documentation/docs/03-loading.md

```diff
@@ -34,7 +30,42 @@ export type LoadOutput<
 	props?: Props;
 	stuff?: Stuff;
 	maxage?: number;
-} | Fallthrough
+}
+
+interface LoadInputExtends {
```

Comment:
I really don't love having all this stuff in the docs — to the average reader it's just noise. Is it really necessary? Hopefully https://github.com/sveltejs/kit/issues/3324 will make it all obsolete anyway

Reply:
Hopefully so, I don't love it either, but this is needed for context to address #3392